packages/app/pages/api/ethpass.ts

```diff
@@ -0,0 +1,59 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+import type { ContractSendMethod } from "web3-eth-contract";
+import axios from "axios";
+import { getWeb3, ipfsAdd } from "../../components/utils";
+import { getAbiFromJson, networkIdByName } from "../../components/constants";
+import passportJson from "@cabindao/nft-passport-contracts/artifacts/contracts/Passport.sol/Passport.json";
+import fs from 'fs';
+
+export default function handler(req: NextApiRequest, res: NextApiResponse) {
+  const { address, network, tokenId, signature, signatureMessage } =
+    typeof req.body === "string" ? JSON.parse(req.body) : req.body;
+  switch (req.method) {
+    case "POST":
+      const web3 = getWeb3(network);
+      const contract = new web3.eth.Contract(
+        getAbiFromJson(passportJson),
+        address
+      );
+      return (contract.methods.symbol() as ContractSendMethod)
+        .call()
+        .then((description) => {
+          const body = {
+            contractAddress: address,
+            chainId: networkIdByName[network],
```

Comment:
@christopherdro Now I'm seeing this:
![image](https://user-images.githubusercontent.com/7143571/155600739-a09c2a79-712a-4b5d-a05c-66432fb17eab.png)

API Response at Thu, 24 Feb 2022 20:15:25 GMT

Reply:
To be clear, this is the error returned from the API call, it returned this error message as an HTML string

Reply:
Ignore this comment, I forgot the URL change